pkg/dns/aws/dns.go

```diff
@@ -102,12 +103,26 @@ func NewProvider(config Config, operatorReleaseVersion string) (*Provider, error
 		return nil, fmt.Errorf("region is required")
 	}
 
+	// TODO: This API will only return hostedzone resources (which are global)
+	// when the region is forced to us-east-1. We don't yet understand why.
```

Comment:
so this comment was for resourcetagging api https://github.com/openshift/cluster-ingress-operator/pull/361/files#diff-e30b09d9b7476806ee0e9a9de47a7d75L108-L110 and the current setup makes it look like it's for r53Config...

Reply:
done